src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -4,26 +4,26 @@
 
 #include "software/simulated_tests/non_terminating_validation_functions/robots_avoid_ball_validation.h"
 #include "software/simulated_tests/non_terminating_validation_functions/robots_slow_down_validation.h"
-#include "software/simulated_tests/simulated_play_test_fixture.h"
+#include "software/simulated_tests/simulated_er_force_sim_play_test_fixture.h"
 #include "software/simulated_tests/validation/validation_function.h"
 #include "software/test_util/test_util.h"
 #include "software/time/duration.h"
 #include "software/world/world.h"
 
-class StopPlayTest : public SimulatedPlayTestFixture
+class StopPlayTest : public SimulatedErForceSimPlayTestFixture
 {
    protected:
     StopPlayTest() : stop_play_rules(initStopPlayRules()) {}
 
     std::vector<ValidationFunction> stop_play_rules;
 
-
     std::vector<ValidationFunction> initStopPlayRules()
     {
         return {
             [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-                // Wait 2 seconds for robots that start too close to the ball to move away
-                if (world_ptr->getMostRecentTimestamp() >= Timestamp::fromSeconds(2))
+                // Wait 4 seconds for robots that start too close to the ball to move away
+                // with ER Simulator
+                if (world_ptr->getMostRecentTimestamp() >= Timestamp::fromSeconds(4))
```

Comment:
Although the validation for this test is passing now, the behaviour of the play is still broken. However, #2571 seems to fix it when I tried it locally.
Current behaviour: 
![halt_play_test_bug](https://user-images.githubusercontent.com/28585597/162598271-6110f769-8a97-4989-ba67-956b017b5de3.gif)